src/ol_infrastructure/applications/edxapp/__main__.py

```diff
@@ -1338,6 +1338,17 @@ def cloud_init_user_data_func(
             first_byte_timeout=60000,  # noqa: WPS432
             between_bytes_timeout=15000,  # noqa: WPS432
         ),
+        fastly.ServiceVclBackendArgs(
+            address=web_lb.dns_name,
+            name="AWS ALB for edX Studio",
+            port=DEFAULT_HTTPS_PORT,
+            ssl_cert_hostname=edxapp_domains["cms"],
+            ssl_sni_hostname=edxapp_domains["cms"],
```

Comment:
Should this be `lms`?

Reply:
No, because we need to add CMS as a backend for Fastly so that we can serve studio MFEs through that layer.

Reply:
But thank you for pointing that out because I was using the wrong key. Just force-pushed to go from cms -> studio